apisix/core/utils.lua

```diff
@@ -335,6 +336,21 @@ end
 _M.resolve_var = resolve_var
 
 
+function _M.get_last_index(str, key)
```

Comment:
for such a basic string operation, I suggest reusing the `pl.stringx` from Penlight library. After all, we already import this lib in our code base. Never reinvent the wheel unless you have to.

https://lunarmodules.github.io/Penlight/libraries/pl.stringx.html#rfind

Example:

```
> print(require("pl.stringx").rfind("apisix__1928", "__"))
7
```

Reply:
Done